petabvis/bar_row.py

```diff
@@ -50,6 +62,15 @@ def get_sd(self):
 
         return sd
 
+    def get_replicate_sd(self):
+        sds = []
+        for replicate in self.replicates:
+            variable = self.get_y_variable_name()
```

Comment:
```suggestion
        variable = self.get_y_variable_name()
        for replicate in self.replicates:
```
Can be moved outside the loop I guess, as in `get_replicate_y_data`